ocs_ci/ocs/pgsql.py

```diff
@@ -338,6 +342,55 @@ def respin_pgsql_app_pod(self):
             resource=app_pod, state=constants.STATUS_RUNNING, timeout=300
         )
 
+    def get_node_utilization(self, node_types=('worker',)):
+        """
+
+        Check CPU and Memory usage and print a table of the data.
+
+        Args:
+            master (bool): if True, check check CPU and Memory usage on master nodes
+            worker (bool): if True, check check CPU and Memory usage on worker nodes
+
+        """
+        usage_memory_table = PrettyTable()
+        usage_memory_table.field_names = ["Node Name", "CPU USAGE adm_top", "CPU USAGE oc_describe",
+                                          "Memory USAGE adm_top", "Memory USAGE oc_describe"]
+
+        for node in node_types:
+            get_adm = get_node_resource_utilization_from_adm_top(node_type=node)
+            get_oc_describe = get_node_resource_utilization_from_oc_describe(node_type=node)
+            for node_master in get_adm:
```

Comment:
would you like to change the var name here (node_master)? 
This name may not fit if we pass  worker